metadata-service/restli-client/src/main/java/com/linkedin/common/client/BaseClient.java

```diff
@@ -66,8 +69,6 @@ protected <T> Response<T> sendClientRequest(
 
   @Override
   public void close() {
-    if (_client != null) {
-      _client.shutdown(new FutureCallback<>());
-    }
+    _client.shutdown(new FutureCallback<>());
```

Comment:
the client is created in init method with `Objects.requireNonNull` so this cannot be null

Reply:
thank you!